code/game/objects/items/devices/multitool.dm

```diff
@@ -96,4 +96,17 @@
 	icon = 'icons/obj/abductor.dmi'
 	icon_state = "multitool"
 	toolspeed = 0.1
-	origin_tech = "magnets=5;engineering=5;abductor=3"
\ No newline at end of file
+	origin_tech = "magnets=5;engineering=5;abductor=3"
+
+/obj/item/device/multitool/abductor/syndi

```

Comment:
https://tgstation13.org/phpBB/viewtopic.php?f=9&t=30&p=313905#p313831

I know your PR history, although you make mostly sprite, map, and numerical changes, you have done things a bit more challenging code-wise as well. This is not beyond your ability, you simply have not given it a shot yet.